docs/settings/devices/cgm.md

```diff
@@ -3,27 +3,27 @@
 CGM selection includes the following:
 
 * Nightscout
-* xDrip
+* xDrip4iOS
 * Dexcom G5
-* Dexcom G6/ONE
-* Dexcom G7/ONE+
+* Dexcom G6 / ONE
+* Dexcom G7 / ONE+
 * Glucose Simulator
 * Libre Transmitter
 * Medtronic Enlite
 
 ## Nightscout
 While using Nightscout as a CGM is an option, it should be avoided if possible because it will not keep Trio running in the background like other CGM options. You will have to open Trio manually to make it run loop cycles.
 
-## xDrip (xDrip4iOS)
+## xDrip4iOS
 To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../operate/build.md#xdrip4ios-as-cgm-source)
 
 However, if you are using Dexcom G6 or ONE with xDrip4iOS, you can choose the Dexcom G6 option in Trio rather than xDrip4iOS, and Trio will intercept the glucose readings even if you're using Shuggah or someone else's TestFlight of xDrip4iOS.
 
-## Dexcom G5/G6 (and ONE)
+## Dexcom G5 / G6 ONE
```

Comment:
I also suggest using `/` between `G6` and  `ONE` to be consistent.

Reply:
Good catch. Thanks. Fix has been pushed.